sdk/nodejs/proto/resource_pb.js

```diff
@@ -1252,7 +1252,7 @@ proto.pulumirpc.RegisterResourceRequest.toObject = function(includeInstance, msg
     customtimeouts: (f = msg.getCustomtimeouts()) && proto.pulumirpc.RegisterResourceRequest.CustomTimeouts.toObject(includeInstance, f),
     deletebeforereplacedefined: jspb.Message.getBooleanFieldWithDefault(msg, 18, false),
     supportspartialvalues: jspb.Message.getBooleanFieldWithDefault(msg, 19, false),
-    remote: jspb.Message.getBooleanFieldWithDefault(msg, 21, false)
+    remote: jspb.Message.getBooleanFieldWithDefault(msg, 20, false)
```

Comment:
I regenerated the protobuf files because they were stale after changing the `remote` field from 21 to 20 per https://github.com/pulumi/pulumi/pull/5280/files/9ff12da5cc6477e8f3b4ed17115b1d9bde1e9c39#r483125014